### ISSUE TITLE
Autoplay not working in ios safari

### ISSUE THREAD

**Issue by Nandhan-augment** [smnbbrv/ngx-plyr#17]
Autoplay is not working in ios safari(12+). But works fine in macOS safari. Is there any option i need to enable it for this issue?

**Comment by smnbbrv** [comment]
That probably should be adressed at https://github.com/sampotts/plyr

**Comment by Nandhan-augment** [comment]
Hi, I have added autoplay attribute in plyr option but not working. the issue is only in safari mobile

**Comment by Jannnnnn** [comment]
Thats not a bug of this repo and not a bug of plyr. It's a security "feature" of safari. You can't autoplay  a video with audio without user interaction before. You can autoplay if the video is muted by `<video autoplay muted ...>` 

See https://webkit.org/blog/6784/new-video-policies-for-ios/

**Comment by Nandhan-augment** [comment]
Yeah heard of that also but my issue is with the audio player, not the video player.

**Comment by smnbbrv** [comment]
Hem, could be my fault here.

Currently the audio / video is delegated to the `<video>` tag regardless of the type.

There was no known issue because of that before; this seems to hit the bulls eye.

I guess I need to patch the lib to respect the type and use `<audio>` for the corresponding type.
